package.json

```diff
@@ -4,29 +4,34 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "prebuild": "prebuild": "npm run clean && npm run lint && echo \"building...\"",
```

Comment:
extra `"prebuild":` 😄

Reply:
Let's also change to `lint:fix` which clears up basic formatting issues that are causing trouble when creating a docker image on certain systems.